homeassistant/components/switch/rpi_rf.py

```diff
@@ -27,8 +27,10 @@
 DEFAULT_SIGNAL_REPETITIONS = 10
 
 SWITCH_SCHEMA = vol.Schema({
-    vol.Required(CONF_CODE_OFF): cv.positive_int,
-    vol.Required(CONF_CODE_ON): cv.positive_int,
+    vol.Required(CONF_CODE_OFF):
```

Comment:
Either the yaml list or `vol.All(cv.ensure_list_csv, [cv.positive_int])` (thanks @andrey-git) is much cleaner and more readable than the regex, I would recommend going with one of those.